packages/mobile/src/navigator/Navigator.tsx

```diff
@@ -140,6 +148,7 @@ const verificationScreens = (Navigator: typeof Stack) => {
       <Navigator.Screen
         name={Screens.VerificationLoadingScreen}
         component={VerificationLoadingScreen}
+        options={headerWithCancelButton}
```

Comment:
I believe we want to actually cancel verification here. There's a `handleBackButton` in `VerificationLoadingScreen` which I think you want to look at

Reply:
Oh I see, for some reason I thought the listener would take care of that but I see now that it's only for the hardware back button. Will revert the changes here and replace with `noHeader`